uptime-service-backend/templates/deployment.yaml

```diff
@@ -7,11 +7,13 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Release.Name }}
+  name: {{ include "common.names.fullname" . }}
```

Comment:
@Smorci We don't need all these helpers taken from bitnami (`_affinities.tpl` , `_labels.tpl`, `_names.tpl`, `_tplvalues.tpl`). We should reuse the default `_helpers.tpl` provided as part of the `helm create` command that already cover this

```
metadata:
  name: {{ include "uptime-service-backend.fullname" . }}
  labels:
    {{- include "uptime-service-backend.labels" . | nindent 4 }}
spec:
```

Reply:
The template generated by the `helm create` already assess all of these questions including `affinity`, `tolerations`, `nodeSelector`, `labels` and `annotations`
e.g. https://github.com/MinaFoundation/helm-charts/blob/main/node-stats-collector/templates/deployment.yaml#L65-L75 and https://github.com/MinaFoundation/helm-charts/blob/main/node-stats-collector/templates/deployment.yaml#L65-L75

On top of this, all our helm charts use the default helm implementation and the `uptime-service-backend` is no different from them.
There is no benefit in adding complexity and divergence to this project

Reply:
That looks quite nicer than the bitnami one. In that case I will use the default one because it is simpler. Good catch, I will get to it.

Reply:
The work related to this refactoring will be present on [this branch](https://github.com/MinaFoundation/helm-charts/tree/PM-1179-refactor-helpers-uptime-service-backend)